spectrum/input.py

```diff
@@ -120,6 +120,20 @@ def create_blog_article(self, title, text='Lorem ipsum', image=None):
         # but in checks module
         # TODO: return id and/or node id
 
+    def create_job_advert(self, title, text='Lorem ipsum'):
+        create_url = "%s/node/add/job_advert" % self._host
+        create_page = self._browser.get(create_url)
+        form = mechanicalsoup.Form(create_page.soup.form)
+        form.input({'title[0][value]': title})
+        form.textarea({'field_job_advert_role_summary[0][value]': text})
+
+        LOGGER.info("Saving form")
+        # not sure why, but `data` here is necessary
```

Comment:
Allows for multiple submit buttons: https://stackoverflow.com/a/547824

Reply:
`self._browser.submit()` sounds like it's submitting the form directly rather than pressing a button.